### DISCUSSION TITLE
Not able to open Web-UI

### DISCUSSION THREAD

**Discussion by Jasper-itsme** [hassio-addons/addon-unifi#416] [General]
I just installed and started the add-on. The log shows no error however when opening the Web-UI I get an "ERR_CONNECTION_TIMED_OUT" message. 

I am connecting to HA via DuckDNS. Perhaps that might cause this? I am however to unexperienced to be able to point out the real rootcause of this behavior.

Is there anyone that has experience with this and faced it?

**Comment by AlecRust** [comment]
This add-on does not support ingress. If you are accessing your Home Assistant remotely via DuckDNS, the web UI will not be exposed. Try it locally on your home network.

Once you've got it all up and running at home, you can enable remote access within the UniFi settings, and access it remotely at https://network.unifi.ui.com/.

**Reply by Jasper-itsme** [reply]
Thanks for the info. I was able to open it via the local url of HA and restore a backup of my windows instance to the HA instance.
